README.md

```diff
@@ -4,5 +4,101 @@
 [![CI](https://github.com/AIRCentre/SARProcessing.jl/actions/workflows/CI.yml/badge.svg)](https://github.com/AIRCentre/SARProcessing.jl/actions/workflows/CI.yml)
 [![Codecov](https://codecov.io/gh/AIRCentre/SARProcessing.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/AIRCentre/SARProcessing.jl)
 
+<p align="center">
+<img src="/Users/igaszczesniak/JuliaEO/notebooks/hands_on_sessions/Working_with_SAR_and_InSAR_Data/SARProcessing/figures/detect_ship.png" height="200"><br>
```

Comment:
This is the absolute path to the file on your computer. It should probably be the relative path

`<img src="figures/detect_ship.png" height="200"><br>`

The image is missing right now when you view the README on github. https://github.com/AIRCentre/SARProcessing.jl/tree/igaszczesniak-patch-1